charts/smokey/templates/cronjob.yaml

```
@@ -6,7 +6,7 @@ spec:
   concurrencyPolicy: Allow
   failedJobsHistoryLimit: 3
   successfulJobsHistoryLimit: 3
-  schedule: "*/10 * * * *"
+  schedule: {{ .Values.cronSchedule }}
```

I think the validator is complaining because of the spaces in the substituted value here. Enclosing in quotes should fix it, e.g.:
```suggestion
  schedule: {{ .Values.cronSchedule | quote }}
```

---

Thanks for fixing this btw!